documentation/src/main/asciidoc/topics/proc_xsite_configuring_jgroups.adoc

```diff
@@ -0,0 +1,73 @@
+[id='configure_relay-{context}']
+= Configuring Cluster Transport for Cross-Site Replication
+Add JGroups RELAY2 to your transport layer so that {brandname} clusters can
+communicate across sites.
+
+This procedure explains how to add a RELAY2 stack definition to
+`infinispan.xml` and configure {brandname} cluster transport.
+
+.Procedure
+
+. Add a stack definition named "xsite" that extends the default JGroups UDP stack.
++
+[source,xml,options="nowrap",subs=attributes+]
+----
+<jgroups>
+   <stack name="xsite" extends="udp">
+   </stack>
+</jgroups>
+----
++
+. Add `RELAY2` to the stack and the local site name.
++
+[source,xml,options="nowrap",subs=attributes+]
+----
+<jgroups>
+   <stack name="xsite" extends="udp">
+      <relay.RELAY2 site="LON" xmlns="urn:org:jgroups"/>
+   </stack>
+</jgroups>
+----
++
+. Create a global cluster that includes all site names.
++
+[source,xml,options="nowrap",subs=attributes+]
+----
+<jgroups>
+   <stack name="xsite" extends="udp">
+      <relay.RELAY2 site="LON" xmlns="urn:org:jgroups"/>
+      <remote-sites>
+         <remote-site name="LON"/>
+         <remote-site name="NYC"/>
+      </remote-sites>
+   </stack>
+</jgroups>
+----
++
+. Add the `default-stack` attribute and specify the JGroups stack that {brandname} clusters use to discover each other and communicate across the cluster. The following example uses the default JGroups stack.
++
+[source,xml,options="nowrap",subs=attributes+]
+----
+<jgroups>
+   <stack name="xsite" extends="udp">
+      <relay.RELAY2 site="LON" xmlns="urn:org:jgroups"/>
+      <remote-sites default-stack="tcp">
```

Comment:
@pruivo and @belaban here we have two protocols. `<stack ... extends="udp">` and `<remote-sites default-stack="tcp">`

can you please explain how UDP is used here? the default JGroups TCP is used for RELAY2, right?

Reply:
it is just reading the UDP stack and adding the RELAY2 protocol. 
So, the intra-cluster communication will use UDP + RELAY2
The site masters will use the TCP stack to communicate between them

Reply:
thanks a mill @pruivo